integration_test/third_party_apps_data/applications/aerospike/metadata.yaml

```diff
@@ -0,0 +1,99 @@
+app_url: "https://aerospike.com/"
+short_name: Aerospike
+long_name: Aerospike
+logo_path: ""
```

Comment:
I think Mary @mtabasko will know what to set for this field :)

Reply:
Let's go with "/stackdriver/images/integrations/aerospike.png"

(I don't have anything there yet, but that's where Evan's team decided they want logos, so we'll go with that.)

Reply:
Added this